### PR TITLE
Printer Health Check RPC (routes and tests)

### PR DESCRIPTION
**Pull request by Annabel1054**

### CODE REVIEW

File: client/api/2DPrinting.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 const {
-  OK, BAD_REQUEST, NOT_FOUND, LED_SIGN_IP
+  OK, BAD_REQUEST, NOT_FOUND
 } = require('../config/config');
 const { addSignLog } = require('../util/logging-helpers');
 
+router.post('/Printer/healthCheck', (req, res) => {
+  healthCheck()
+    .then(response => {
+      console.log(response);
```
Comment (evanugarte) [inline client/api/2DPrinting.js:12]:
```suggestion
```

File: proto/print.proto [hunk 2]
```diff
@@ -3,15 +3,24 @@ syntax = "proto3";
...
+    string message = 1;
+}
+
+message HealthCheckUpRequest {
+    string member_name = 1;
+}
+
+message PrinterHealthStatus {
     string message = 1;
 }
```
Comment (evanugarte) [inline proto/print.proto:26]: add new line to bottom of this file

File: proto/print.proto [hunk 3]
```diff
@@ -3,15 +3,24 @@ syntax = "proto3";
 
 service Printer {
     rpc PrintPage (PrintRequest) returns (PrintResponse);
+    rpc HealthCheck (HealthCheckUpRequest) returns (PrinterHealthStatus);
```
Comment (evanugarte) [inline proto/print.proto:6]:
```suggestion
    rpc HealthCheck (PrinterHealthCheckRequest) returns (PrinterHealthStatus);
```

File: proto/print.proto [hunk 4]
```diff
@@ -3,15 +3,24 @@ syntax = "proto3";
...
     map<string, string> options = 3;
     bytes encoded_file = 4;
 }
 
-message PrintResponse{
+message PrintResponse {
+    string message = 1;
+}
+
+message HealthCheckUpRequest {
```
Comment (evanugarte) [inline proto/print.proto:20]:
```suggestion
message PrinterHealthCheckRequest {
```

Comment (evanugarte) [reply proto/print.proto:20]: btw youll need to regenerate after changing this

File: client/printing/print_grpc_pb.js [hunk 5]
```diff
@@ -4,6 +4,17 @@
 var grpc = require('grpc');
 var print_pb = require('./print_pb.js');
```
Comment (hsorathia) [inline client/printing/print_grpc_pb.js:5]: while you're working on this can you change instances of var over to `const` or `let`

Comment (evanugarte) [reply client/printing/print_grpc_pb.js:5]: i mentioned this before, this is auto generated code. nobody should touch this

Comment (hsorathia) [reply client/printing/print_grpc_pb.js:5]: ahhh sorry, i think i missed it

**Comment by evanugarte** [comment]
rebase home

**Review by jerrylee17 — APPROVED** [review]

**Review by IsitaB — APPROVED** [review]

**Pull request merged**